src/lib.rs

```diff
@@ -778,6 +809,21 @@ impl Client {
         })
     }
 
+    async fn by_from(
```

Comment:
I'm not sure I follow this name?

Reply:
Ah, sorry, I basically just copied the whole code path for `find`, which uses `Client::by`. Seeing as `by_from` is not used anywhere else, I can just move the code into `find_from`.